docs/xostor/xostor.md

```diff
@@ -856,3 +856,68 @@ Note: iptables config must also be modified to remove LINSTOR port rules (edit `
 For more info:
 - https://linbit.com/blog/linstors-auto-evict/
 - https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-linstor-auto-evict
+
+### How to enable dm-cache (Device mapper cache)?
+
+`dm-cache` is a part of the Linux kernel's device mapper, which is a system for managing storage devices. It lets you use fast storage, like SSDs, as a cache for slower storage, like HDDs. This helps improve performance by creating a hybrid storage system on a XOSTOR SR.
+
+:::warning
+This feature is currently experimental and not covered by [Vates Pro Support](https://vates.tech/pricing-and-support).
+:::
+
+On each host, create a new PV and VG using your cache devices:
+```
+pvcreate linstor_group_cache <CACHE_DEVICES>
+vgcreate linstor_group_cache <CACHE_DEVICES>
```

Comment:
Only `vgcreate` is needed, giving a device to `vgcreate` will automatically call `pvcreate` with the default configuration.